Electron-Basic-Screen-Sharing/V12-to-V16/README.md

```diff
@@ -0,0 +1,188 @@
+session. You must connect before sending or receiving audio-video streams in the session (or before
+interacting with the session in any way). The `connect()` method takes two parameters -- a token
+and a completion handler function:
+
+```javascript
+// Connect to the Session
+session.connect(token, (error) => {
+  
+  // If the connection is successful, initialize a publisher and publish to the session
+  if (error) {
+    return console.log(`There was an error connecting to session: ${error}`);
+  } else {
+    // Create a publisher
+    // Publish
+  }
+});
+```
+
+
+An error object is passed into the completion handler of the `session.connect()` method if the
+client fails to connect to the OpenTok session. Otherwise, no error object is passed in, indicating
+that the client connected successfully to the session.
+
+The Session object dispatches a `streamDestroyed` event when the stream is Destroyed. The application defines an event handler for this event:
+```javascript
+session.on({
+  streamCreated: (event) => {
+    session.subscribe(event.stream, 'subscriber', (error) => {
+      if (error) {
+        console.log(`There was an issue subscribing to the stream ${event}`);
+      }
+    });
+  }
+});
+```
+
+## Publishing an audio video stream to the session
+
+Upon successfully connecting to the OpenTok session (see the previous section), the application
+initializes an OpenTok Publisher object and publishes an audio-video stream to the session. This is
+done inside the completion handler for the connect() method, since you should only publish to the
+session once you are connected to it.
+
+The Publisher object is initialized as shown below. The `OT.initPublisher()` method takes three
+optional parameters:
+
+* The target DOM element or DOM element ID for placement of the publisher video
+* The properties of the publisher
+* The completion handler
+
+```javascript
+const publisherProperties = { videoSource: 'screen' };
+
+const publisher = OT.initPublisher('publisher', publisherProperties, (error) => {
+  if (error) {
+    return console.log(`There was an error initializing the publisher: ${error}`);
+  }
+});
+```
+
+Once the Publisher object is initialized, we publish to the session using the `publish()`
+method of the Session object:
+
+```javascript
+session.publish(publisher, (error) => {
+  if (error) {
+    return console.log(`There was an error when trying to publish: ${error}`);
+  }
+});
+```
+## Subscribing to another client's audio-video stream
+
+The Session object dispatches a `streamCreated` event when a new stream (other than your own) is
+created in a session. A stream is created when a client publishes to the session. The
+`streamCreated` event is also dispatched for each existing stream in the session when you first
+connect. This event is defined by the StreamEvent object, which has a `stream` property,
+representing stream that was created. The application adds an event listener for the
```

Comment:
Should it be "representing *the* stream" ?

Reply:
Yep!